src-tauri/tauri.conf.json

```
@@ -1,387 +1,395 @@
+            "cmd": "fcrackzip",
+            "args": true
+          },
+          {
+            "name": "find",
+            "cmd": "find",
+            "args": true
+          },
+          {
+            "name": "gitleaks",
+            "cmd": "gitleaks",
+            "args": true
+          },
+          {
+            "name": "gobuster",
+            "cmd": "gobuster",
+            "args": true
+          },
+          {
+            "name": "goldeneye",
+            "cmd": "goldeneye",
+            "args": true
+          },
+          {
+            "name": "hashcat",
+            "cmd": "hashcat",
+            "args": true
+          },
+          {
+            "name": "hydra",
+            "cmd": "hydra",
+            "args": true
+          },
+          {
+            "name": "impacket-secretsdump",
+            "cmd": "impacket-secretsdump",
+            "args": true
+          },
+          {
+            "name": "impacket-wmiexec",
+            "cmd": "impacket-wmiexec",
+            "args": true
+          },
+          {
+            "name": "john",
+            "cmd": "john",
+            "args": true
+          },
+          {
+            "name": "kill",
+            "cmd": "kill",
+            "args": true
+          },
+          {
+            "name": "ls",
+            "cmd": "ls",
+            "args": true
+          },
+          {
+            "name": "masscan",
+            "cmd": "masscan",
+            "args": true
+          },
+          {
+            "name": "metagoofil",
+            "cmd": "metagoofil",
+            "args": true
+          },
+          {
+            "name": "msfvenom",
+            "cmd": "msfvenom",
+            "args": true
+          },
+          {
+            "name": "nbtscan",
+            "cmd": "nbtscan",
+            "args": true
+          },
+          {
+            "name": "nikto",
+            "cmd": "nikto",
+            "args": true
+          },
+          {
+            "name": "nslookup",
+            "cmd": "nslookup",
+            "args": true
+          },
+          {
+            "name": "parsero",
+            "cmd": "parsero",
+            "args": true
+          },
+          {
+            "name": "pip3",
+            "cmd": "pip3",
+            "args": true
+          },
+          {
+            "name": "pkexec",
+            "cmd": "pkexec",
+            "args": true
+          },
+          {
+            "name": "python3",
+            "cmd": "python3",
+            "args": true
+          },
+          {
+            "name": "rar2john",
+            "cmd": "rar2john",
+            "args": true
+          },
+          {
+            "name": "rcrack",
+            "cmd": "rcrack",
+            "args": true
+          },
+          {
+            "name": "rtsort",
+            "cmd": "rtsort",
+            "args": true
+          },
+          {
+            "name": "rtgen",
+            "cmd": "rtgen",
+            "args": true
+          }
```

Needs to have "," after the closing "}".

```suggestion
},